pubspec.yaml

```diff
@@ -13,7 +13,11 @@ dependencies:
   cupertino_icons: ^1.0.2
   cln_grpc: ^0.0.1-beta.2
   cln_common: ^0.0.1-beta.1
-  clightning_rpc: 0.0.2-beta.1
+#  clightning_rpc: 0.0.2-beta.1
+  clightning_rpc:
+    git:
+      url: https://github.com/KAVAN-DESAI/lndart.clightning.git
```

Comment:
I'm trying to release the `beta.2`

https://github.com/dart-lightning/lndart.cln/releases/tag/cln_rpc-v0.0.2-beta-2

make a double check on pub.dev tomorrow